chrome/content/zotero/xpcom/zotero.js

```diff
@@ -507,6 +509,35 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 			}
 		}
 		
+		// Check for DB purge
+		var newAccountFile = dataDir.clone();
+		newAccountFile.append('new-account');
```

Comment:
I don't think we should bother users with this sort of thing. If they want a new library, they'll find a way (probably by switching the data directory). And introduces code complexity, since you'll want to show prompt on manual unlink, but not when the API key expires or similar. I do agree with `new-account` to `reset-data-directory` change though.